buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -59,7 +60,7 @@ steps:
   timeoutInMinutes: 10
 
 - bash: |
-    python $(workspace)/buildlib/pr/io_demo/iodemo_analyzer.py -d $(workspace)/${{ parameters.name }} --duration ${{ parameters.duration }}
+    python /hpc/noarch/git_projects/hpc-mtt-conf/scripts/iodemo_analyzer.py --branch $(Build.SourceBranchName) -d $(workspace)/${{ parameters.name }} --duration ${{ parameters.duration }} -t 3
```

Comment:
manager_script_dir ?
what is "-t 3" ?

Reply:
threshold for 'w/o traffic'

Reply:
in minutes?
what was it before this change?

Reply:
was 3min by default, but in mtt repo threshold  is 1min by default